OpenSubtitlesHandler/Models/Responses/Attributes.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace OpenSubtitlesHandler.Models.Responses
+{
+    public class Attributes
+    {
+        public string SubtitleId;
```

Comment:
Why are these fields? It's generally considered bad practice to use public fields. Use auto-properties instead.

Reply:
All fields were changed into properties in d82fb3f72a7f675def39de0bc521b613f1f232ae